src/content/docs/distribute/flatpak.mdx

```diff
@@ -1,45 +1,99 @@
 ---
-title: Flatpak / Flathub
-draft: true
+title: Flathub
 sidebar:
   order: 1
-  badge:
-    text: WIP
-    variant: caution
 ---
 
+import { Tabs, TabItem, Card } from '@astrojs/starlight/components'
+
 For detailed information on how Flatpak works, you can read [Building your first Flatpak](https://docs.flatpak.org/en/latest/first-build.html)
 
 This guide assumes you want to distribute your Flatpak via [Flathub](https://flathub.org/), the most commonly used platform for Flatpak distribution. If you plan on using other platforms, please consult their documentation instead.
-
-TODO: deb vs from source
-
-## Build locally
+### Prerequisites
 
 To test your app inside the Flatpak runtime you can build the Flatpak locally first before uploading your app to Flathub. This can also be helpful if you want to quickly share development builds.
 
 **1. Install `flatpak` and `flatpak-builder`**
 
 To build Flatpaks locally you need the `flatpak` and `flatpak-builder` tools. For example on Ubuntu you can run this command:
+<Tabs>
+  <TabItem label="Debian">
 
 ```sh
+sudo apt update
 sudo apt install flatpak flatpak-builder
 ```
 
+  </TabItem>
+  <TabItem label="Arch">
+
+```sh
+sudo pacman -Syu
+sudo pacman -S --needed flatpak flatpak-builder
+```
+  </TabItem>
+  <TabItem label="Fedora">
+
+```sh
+sudo dnf check-update
+sudo dnf install flatpak flatpak-builder
+```
+  </TabItem>
+  <TabItem label="Gentoo">
+
+```sh
+sudo emerge --ask \
+sys-apps/flatpak \
+dev-util/flatpak-builder
+```
+  </TabItem>
+</Tabs>
+
 **2. Install the Flatpak Runtime**
 
 ```shell
 flatpak install flathub org.Gnome.Platform//46 org.Gnome.Sdk//46
 ```
 
-**3. Build the .deb of your tauri-app**
+**3. [Build the .deb of your tauri-app](https://tauri.app/v1/guides/building/linux#debian)**
 
-TODO: Link to Build guide
+**4. Create the manifest**
 
-**4. Build and install the flatpak**
+```yaml
+id: org.your.id
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
 
-TODO: manifest should probably come first
+command: tauri-app
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --device=dri # OpenGL, not necessary for all projects
+  - --share=ipc
 
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/your_username/your_repository/releases/download/v1.0.1/yourapp_1.0.1_amd64.deb
+        sha256: 08305b5521e2cf0622e084f2b8f7f31f8a989fc7f407a7050fa3649facd61469 # This is required if you are using a remote source
+        only-arches: [x86_64] #This allows you to make your source architecture specific so that arm users dont install amd64 apps. Another acceptable value is [aarch64].
```

Comment:
```suggestion
        only-arches: [x86_64] # This allows you to make your source architecture specific so that arm users dont install amd64 apps. Another acceptable value is [aarch64].
```